cli/src/test/java/org/commonjava/maven/ext/cli/CliTest.java

```diff
@@ -121,18 +124,16 @@ public void checkTargetDefaultMatches() throws Exception
     public void checkLocalRepositoryWithDefaults() throws Exception
     {
         Cli c = new Cli();
-        File settings = writeSettings( temp.newFile());
+        File settings = writeSettings( temp.newFile() );
 
-        TestUtils.executeMethod( c, "run", new Object[] { new String[] { "-s", settings.toString()} } );
+        TestUtils.executeMethod( c, "run", new Object[] { new String[] { "-s", settings.toString() } } );
 
         ManipulationSession session = (ManipulationSession) FieldUtils.readField( c, "session", true );
-        MavenSession ms = (MavenSession)FieldUtils.readField( session, "mavenSession", true );
+        MavenSession ms = (MavenSession) FieldUtils.readField( session, "mavenSession", true );
 
-        assertEquals( ms.getRequest().getLocalRepository().getBasedir(),
-                      ms.getRequest().getLocalRepositoryPath().toString() );
+        assertEquals( ms.getRequest().getLocalRepository().getBasedir(), ms.getRequest().getLocalRepositoryPath().toString() );
         assertEquals( "File " + new File( ms.getRequest().getLocalRepository().getBasedir() ).getParentFile().toString()
-                                      + " was not equal to " + System.getProperty( "user.home" ) + File.separatorChar
-                                      + ".m2",
+                                      + " was not equal to " + System.getProperty( "user.home" ) + File.separatorChar + ".m2",
```

Comment:
Minor, but why does the code use `separatorChar` instead of `separator` if it's concatenating strings?

Reply:
Shrug .... yeah probably could be change but this was pre-existing code.